README.md

```diff
@@ -136,6 +136,10 @@ jobs:
           # installers-regex: '\.exe$' # Only .exe files
 ```
 
+### Alternative actions 🔄
+
+- Run Komac manually: [michidk/run-komac](https://github.com/michidk/run-komac)
```

Comment:
```suggestion
- Release automation, which can be used with external repositories: [michidk/winget-updater](https://github.com/michidk/winget-updater)
- Run Komac manually: [michidk/run-komac](https://github.com/michidk/run-komac)
```

Reply:
@russellbanks Nice approach with the list. Also added my release automation script. What do you think?